packages/polaris-viz/src/components/TooltipContent/components/TooltipRow/TooltipRow.tsx

```diff
@@ -64,6 +66,20 @@ export function TooltipRow({
       >
         {value}
       </span>
+      {trend != null && (
+        <div className={styles.Trend}>
+          <TrendIndicator {...trend} />
+        </div>
+      )}
     </div>
   );
 }
+
+const DEFAULT_COLUMN_COUNT = 2;
+
+export function getColumnCount(data: TooltipData) {
+  const hasTrend = data.data.some(({trend}) => trend != null);
+  const hasColor = data.data.some(({color}) => color != null);
+
+  return DEFAULT_COLUMN_COUNT + (hasColor ? 1 : 0) + (hasTrend ? 1 : 0);
+}
```

Comment:
Would it make sense to define this in `TooltipSeries`? Or maybe a separate util file 🤔

Reply:
Moved this to it's own file!